src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -216,7 +216,7 @@ static <T extends LfElement> void updateRemoteVoltageControlEquations(VoltageCon
                 .map(vc -> equationSystem.getEquation(vc.getControlledBus().getNum(), AcEquationType.BUS_TARGET_V).orElseThrow())
                 .collect(Collectors.toList());
 
-        if (controlledBus.isDisabled() || voltageControl.isDisabledAndAlsoAllItsDependentVoltageControls()) {
```

Comment:
@geofjamg for me, a disabled controlled bus is included in `voltageControl.isDisabledAndAlsoAllItsDependentVoltageControls()`.